lib/config/rpcProviderProdConfig.json

```diff
@@ -1,13 +1,13 @@
 [
   {
     "chainId": 42220,
-    "useMultiProviderProb": 0.1,
+    "useMultiProviderProb": 1,
```

Comment:
I think you need to update your branch, as this was from https://github.com/Uniswap/routing-api/pull/542

Reply:
Yes. Added.